### PR TITLE
Corrected all errors

### PR DESCRIPTION
**Pull request by singhcoder694**
Corrected all error told by you everything is working perfectly fine. 405 error is also gone.... using gmail I have checked for many times now 405 is not there but couldn't try using Zoho since I don't have that account and it is not free to use.

## Summary by CodeRabbit

- **New Features**
  - Unified success messages across different generators for a consistent user experience.

- **Refactor**
  - Updated email authentication credentials to enhance security and maintain environmental variable standards.
  - Refined API response handling to prioritize success messages and streamline user feedback.

- **Bug Fixes**
  - Adjusted HTTP status code responses for confirmation processes to ensure accurate communication with the client.

### CODE REVIEW

**Comment by singhcoder694** [comment]
What I think is due to missing return statements mail was encountering 405 error otherwise it will not, you may try using your zoho id using by locating that in env file with same value as given in code @ank1raj

File: config/nodemailer.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ export const transporter= nodemailer.createTransport({
     secure: true, 
     port: 465, 
     auth: { 
-        user: process.env.EMAIL_SENDER, 
-        pass: process.env.EMAIL_PASSWORD, 
+        user: process.env.EMAIL, 
+        pass: process.env.PASS,
```
Comment (ank1traj) [inline config/nodemailer.js:8]: `EMAIL_PASSWORD` would be a better naming here

**Review by ank1traj — CHANGES_REQUESTED** [review]

File: config/nodemailer.js (hunk 1 above)
Comment (singhcoder694) [reply config/nodemailer.js]: Done @ank1traj

**Review by ank1traj — APPROVED** [review]

**Comment by ank1traj** [comment]
@singhcoder694 No, it's still not fixed. let's sit together tomorrow and work on this and fix this asap.

**Comment by singhcoder694** [comment]
Ok sir, I fixed it properly but if not we can meet to fix it.... but how can we meet? @ank1traj

File: pages/api/confirm.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ export default async function handler(req, res) {
     if (data.value=="Test Case Generated Successfully!!!"){
         await transporter.sendMail({
             from: data.email,
-            to:process.env.EMAIL,
+            to:process.env.EMAIL_SENDER,
```
Comment (ank1traj) [inline pages/api/confirm.js:8]: this was the issue @singhcoder694

Comment (singhcoder694) [reply pages/api/confirm.js:8]: Ohh, env this is the reason on my system it was working properly , due to change in env variable name.... Okk👍

Comment (ank1traj) [reply pages/api/confirm.js:8]: No it should be like this

```
to: data.email,
from:process.env.EMAIL
```

And, we're using noreply mail ID. So, it getting bounced off.

Comment (singhcoder694) [reply pages/api/confirm.js:8]: Got it👍

**Pull request merged**